Replicator/Puller.cc

```diff
@@ -207,13 +225,25 @@ namespace litecore::repl {
     void Puller::maybeStartIncomingRevs() {
         while ( connected() && _activeIncomingRevs < tuning::kMaxActiveIncomingRevs
                 && _unfinishedIncomingRevs < tuning::kMaxIncomingRevs && !_waitingRevMessages.empty() ) {
-            auto msg = _waitingRevMessages.front();
-            _waitingRevMessages.pop_front();
-            if ( _waitingRevMessages.empty() ) {
-                Signpost::end(Signpost::revsBackPressure);
-                logVerbose("Back pressure ended for changes messages");
+            // Waiting Rev is either regular rev message (MessageIn) or revoked rev (RevToInsert)
+            auto waiting = _waitingRevMessages.front();
+            if ( waiting.index() == 0 ) {
```

Comment:
std::get_if  appears more readable.

Reply:
Will take a look

Reply:
Changed to use this - agree is more readable